compiler/src/test.rs

```diff
@@ -118,6 +118,7 @@ impl Namespace for CompileNamespace {
         let parsed = parse_program(
             &test.content,
             Some(AstSnapshotOptions {
+                spans_enabled: true,
```

Comment:
We ideally want this disabled for the tests. Span's aren't something we are really testing/care about here. No need to create another version of the test expectations.

Reply:
Sounds good!